images/image-user/Makefile-orig

```diff
@@ -0,0 +1,45 @@
+# Copyright 2018 The Kubernetes Authors.
```

Comment:
I'm wondering why we should keep that file.

Reply:
Good point, just left it in incase for reference. Will remove it in next commit.

Reply:
Thanks! :pray:

Reply:
Done